save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationSettingsMenu.kt

```diff
@@ -67,6 +72,23 @@ external interface OrganizationSettingsMenuProps : Props {
     var onCanCreateContestsChange: (Boolean) -> Unit
 }
 
+/**
+ * Makes a call to change project status
+ *
+ * @param organizationName name of the organization whose status will be changed
+ * @param status is new status
+ * @return response
+ */
+fun responseChangeOrganizationStatus(status: OrganizationStatus, organizationName: String): suspend WithRequestStatusContext.() -> Response = {
```

Comment:
I'd change the order of parameters to match that in the API URL:

```suggestion
fun responseChangeOrganizationStatus(organizationName: String, status: OrganizationStatus): suspend WithRequestStatusContext.() -> Response = {
```

Reply:
Done